ViewGenerator/tools/webpack/Rules/Files.ts

```diff
@@ -1,84 +1,44 @@
-﻿import { basename } from "path";
-import { RuleSetRule } from "webpack";
+﻿import { AssetInfo, NormalModule, PathData, RuleSetRule } from "webpack";
```

Comment:
will these changes fix the hack we have on the framework to have the web components packages available in that project?

Reply:
If so, I haven't changed that

Reply:
<img width="265" alt="image" src="https://github.com/OutSystems/ReactView/assets/20876378/2758f511-6e3b-4b80-ab0e-a5074ba80359">
after the pnpm we had to have direct references on the package.json of the Framework because of this File.ts

Reply:
we need to test this, I'm not convinced it will fix it..

Reply:
Yep, I do not think it will fix it... But I have some ideas that we can leverage afterwards